pulp_container/app/registry_api.py

```diff
@@ -519,6 +519,9 @@ def get_queryset(self, *args, **kwargs):
         distribution_permission = "container.pull_containerdistribution"
         namespace_permission = "container.namespace_pull_containerdistribution"
 
+        if settings.get("TOKEN_AUTH_DISABLED", True):
```

Comment:
This is dangerous because if the setting is not defined then it will se set to True

Do instead `if settings.get("TOKEN_AUTH_DISABLED", False)`